react-native-sdk/index.js

```diff
@@ -0,0 +1,89 @@
+import 'react-native-gesture-handler';
```

Comment:
I see there is some duplicity here with index.native.js, how about moving the root component in that file to some Root.js and then importing it here and there?

Reply:
We're using a ref in the sdk but not in the index.native. In my opinion the reduction of duplications is worth less than the implementation of another ref forwarding component that also only sometimes needs the ref.

Reply:
OK, let's leave it for now and maybe we come up with other ideas.